### PR TITLE
Redirect to login page on 401

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: kotsadm/pkg/handlers/logout.go [hunk 1]
```diff
@@ -11,22 +11,30 @@ import (
 func Logout(w http.ResponseWriter, r *http.Request) {
 	sess, err := session.Parse(r.Header.Get("Authorization"))
 	if err != nil {
+		if store.GetStore().IsNotFound(err) {
+			JSON(w, http.StatusNoContent, "")
```
Comment (emosbaugh) [inline kotsadm/pkg/handlers/logout.go:15]: doesnt this return content? why not `w.WriteHeader(http.StatusNoContent)`?

File: kotsadm/pkg/handlers/logout.go [hunk 2]
```diff
@@ -11,22 +11,30 @@ import (
...
+			return
+		}
+		if r.Header.Get("Authorization") == "" {
+			JSON(w, http.StatusNoContent, "")
+			return
+		}
 		logger.Error(err)
```
Comment (marccampbell) [inline kotsadm/pkg/handlers/logout.go:22]: @divolgin how would you feel about not logging this. It's common when switching clusters, so it's confusing to see this error in logs.

File: kotsadm/pkg/apiserver/server.go [hunk 3]
```diff
@@ -102,6 +101,11 @@ func Start() {
 	* Session auth routes
 	**********************************************************************/
 
+	sessionAuthQuietRouter := r.PathPrefix("").Subrouter()
+	sessionAuthQuietRouter.Use(handlers.RequireValidSessionQuietMiddleware)
+
+	sessionAuthQuietRouter.Path("/api/v1/ping").Methods("GET").HandlerFunc(handlers.Ping)
```
Comment (emosbaugh) [inline kotsadm/pkg/apiserver/server.go:107]: can you remove this comment? https://github.com/replicatedhq/kots/blob/master/kotsadm/pkg/handlers/ping.go#L18

File: kotsadm/pkg/handlers/logout.go [hunk 4]
```diff
@@ -11,22 +11,30 @@ import (
 func Logout(w http.ResponseWriter, r *http.Request) {
 	sess, err := session.Parse(r.Header.Get("Authorization"))
 	if err != nil {
+		if store.GetStore().IsNotFound(err) {
+			JSON(w, http.StatusNoContent, "")
+			return
+		}
+		if r.Header.Get("Authorization") == "" {
```
Comment (emosbaugh) [inline kotsadm/pkg/handlers/logout.go:18]: why not move this to the top before session.Parse?

File: kotsadm/pkg/handlers/logout.go (hunk 2 above)
Comment (divolgin) [reply kotsadm/pkg/handlers/logout.go:22]: if you switch clusters, it should hit the store's IsNotFound

**Pull request merged**